locale/fi/LC_MESSAGES/django.po

```diff
@@ -2071,21 +2071,26 @@ msgstr "Palautuksesi"
 msgid "Points required to pass"
 msgstr "Pistevaatimus"
 
-#: exercise/templates/exercise/_exercise_info.html:53
-#: exercise/templates/exercise/exercise_plain.html:103
+#: exercise/templates/exercise/_exercise_info.html:55
+#: exercise/templates/exercise/exercise_plain.html:114
 msgid "Deadline"
 msgstr "Määräaika"
 
-#: exercise/templates/exercise/_exercise_info.html:57
-#: exercise/templates/exercise/exercise_plain.html:98
+#: exercise/templates/exercise/_exercise_info.html:58
+#: exercise/templates/exercise/exercise_plain.html:97
+msgid "Extended deadline"
+msgstr "Pidennetty palautusaika"
```

Comment:
"Deadline" has been translated to "määräaika" previously, so we should use the same word here ("pidennetty palautusaika" is changed to "pidennetty määräaika").